src/control/KinematicSimulationTrajectoryExecutor.cpp

```diff
@@ -106,7 +106,7 @@ void KinematicSimulationTrajectoryExecutor::step()
   }
 
   auto timeSinceBeginning = system_clock::now() - mExecutionStartTime;
-  double t = duration_cast<seconds>(timeSinceBeginning).count();
+  auto t = duration<double>(timeSinceBeginning).count();
```

Comment:
Nit: I believe the unit of time here is seconds. It would be nice to either (1) change the variable name to encode the unit or (2) add trailing comment something like `// sec`.

Additionally, it would be much better to comment the unit of the time parameter to the docstring of `Trajectory::evaluate()`.